src/rgw/setup/rgw.py

```diff
@@ -700,10 +707,20 @@ def _update_svc_data_path_value(conf: MappedConf, client_section: str):
         "Update svc config file with data path key which needs pre-processing values incase of default values."
         # Fetch cluster-id
         cluster_id = Rgw._get_cortx_conf(conf, const.CLUSTER_ID_KEY)
+        svc_config_file = Rgw._get_rgw_config_path(conf)
+        Rgw._load_rgw_config(Rgw._conf_idx, f'ini://{svc_config_file}')
```

Comment:
put this 'ini' config store also in some constant, so tomorrow we can plan to change it to consul and use directly

Reply:
done